Mathlib/Analysis/CStarAlgebra/CStarMatrix.lean

```diff
@@ -439,9 +439,8 @@ def mapₙₐ [Fintype n] [Semiring R] [NonUnitalNonAssocSemiring A] [Module R A
   map_add' := by simp [map_add]
   map_mul' M N := by
     ext
-    simp only [mapₗ_apply, map, LinearMap.coe_coe, ofMatrix_apply]
-    simp_rw [mul_apply, map_sum, map_mul]
-    rfl
+    simp only [mapₗ_apply, map, LinearMap.coe_coe, ofMatrix_apply, mul_apply, map_sum, map_mul,
```

Comment:
Pre-existing, but: does un-squeezing this simp result in a performance difference?

Reply:
Pretty noisy, but yes, it seems like it might be on the order of half a second / 30% slower on my machine.

Reply:
Would you like to add a comment then?